openstack/images_image_v2.go

```diff
@@ -100,10 +100,28 @@ func resourceImagesImageV2File(d *schema.ResourceData) (string, error) {
 				return "", fmt.Errorf("Error creating file %q: %s", filename, err)
 			}
 			defer file.Close()
-			resp, err := http.Get(furl)
+			client := &http.Client{}
```

Comment:
you haven't used the provider's http client. You need to pass the `imageClient` pointer to the func, then re-use its client like:
```suggestion
			client := serviceClient.ProviderClient.HTTPClient
```

Reply:
done.